dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/Helper.java

```diff
@@ -902,9 +459,9 @@ public static boolean isGit(String url) {
      * @param client
      * @param objectMapper
      * @param tokenDAO
-         * @param userId
-         * @return
-         */
+     * @param userId
+     * @return
+     */
     public static Boolean checkIfUserOwns(final Tool tool,final HttpClient client, final ObjectMapper objectMapper, final TokenDAO tokenDAO, final long userId) {
```

Comment:
Not strictly part of this pull request, but let's see if this can be changed to `boolean`.